### PR TITLE
adds getMediaUrl changes to score core, creator core and player core

### PR DESCRIPTION
**Pull request by iturgeon**

### CODE REVIEW

File: _developers/creator-core.md [hunk 1]
```diff
@@ -67,6 +67,14 @@ Materia.CreatorCore.showMediaImporter(['jpg', 'gif', 'png'])
 Materia.CreatorCore.showMediaImporter(['mp3'])
 ```
 
+## Materia.CreatorCore.getMediaUrl
+
+Convert a materia asset id into a url.
```
Comment (FrenjaminBanklin) [inline _developers/creator-core.md]: Capitalize 'Materia'?

File: _developers/score-core.md [hunk 2]
```diff
@@ -69,3 +69,11 @@ Adjusts the height of the score screen in pixels.
 ```javascript
 Materia.ScoreCore.setHeight(350)
 ```
+
+## Materia.ScoreCore.getMediaUrl
+
+Convert a materia asset id into a url.
```
Comment (FrenjaminBanklin) [inline _developers/score-core.md]: Capitalize 'Materia'?

**Review by FrenjaminBanklin — CHANGES_REQUESTED** [review]
Would suggest always capitalizing 'Materia' outside of code blocks, looks okay otherwise.

File: _developers/score-core.md (hunk 2 above)
Comment (iturgeon) [reply _developers/score-core.md]: Good idea, I did a global sweep for lower case materias

**Review by FrenjaminBanklin — APPROVED** [review]

**Pull request merged**